recipes/base_install.rb

```diff
@@ -86,12 +86,14 @@
 # Manage SSH via Chef
 include_recipe "openssh"
 
-# Install SSH target checker
-cookbook_file 'ssh_target_checker.sh' do
-  path "/usr/local/bin/ssh_target_checker.sh"
-  owner "root"
-  group "root"
-  mode "0755"
+unless node['platform'] == 'centos' && node['platform_version'].to_i < 7
```

Comment:
Optional: We could also install this everywhere to avoid additional custom logic

Reply:
ok